### PR TITLE
Add coverage of Org settings

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/382991822377058

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://42-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://42-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://42-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://42-bb-docs.surge.sh

File: integrations/default_settings/README.adoc [hunk 1]
```diff
@@ -0,0 +1,153 @@
+= Default integration settings
```
Comment (benedict-jw) [inline integrations/default_settings/README.adoc]: Integration settings for all apps

File: integrations/default_settings/README.adoc [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Each of the available link:../README.adoc[integration] settings are
+app-specific. This section describes how you can specify default
```
Comment (benedict-jw) [inline integrations/default_settings/README.adoc]: remove 'default'

File: integrations/default_settings/README.adoc [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+integration settings that automatically apply to new apps, or that can
+be applied to any applications that already exist within an
+organization.
+
+[IMPORTANT]
+Default integration settings are available for the following
```
Comment (benedict-jw) [inline integrations/default_settings/README.adoc]: Integration settings that can be applied to all apps are available for:

File: integrations/default_settings/README.adoc (hunk 1 above)
Comment (benedict-jw) [reply integrations/default_settings/README.adoc]: Watch use of 'default' in the org scenario throughout the doc

**Comment by bbdocs** [comment]
See the website here: http://42-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://42-bb-docs.surge.sh

**Pull request merged**